src/plugins/intel_gpu/tests/unit/test_cases/permute_gpu_test.cpp

```diff
@@ -1977,3 +2057,183 @@ TEST_P(permute_tile_fsv_5d, i64_cached) {
     auto p = GetParam();
     run_test<cldnn::data_types::i64>(p.sizes, p.format_fsv, "permute_tile_8x8_4x4_fsv", {}, true);
 }
+
+class permute_f_y_axes_tile: public TiledPermuteTest {};
+
+INSTANTIATE_TEST_SUITE_P(smoke_permute_f_y_axes_tile,
+                         permute_f_y_axes_tile,
+                         ::testing::ValuesIn(std::vector<TiledPermuteParam>{
+                             {{1, 4, 8, 1}, format::bfyx},                // permute_f_y_axes
+                             {{1, 4, 8, 1}, format::b_fs_yx_fsv16},       // permute_f_y_axes
+                             {{1, 32, 256, 512}, format::b_fs_yx_fsv32},  // THREE_DIM_TRANSPOSE with subgroup
+                             {{1, 32, 256, 512}, format::bfyx},           // PERMUTE_SIMPLE_MEM_COPY
+                             {{1, 256, 256, 1}, format::b_fs_yx_fsv32},   // permute_f_y_axes
+                             {{1, 8, 4, 4}, format::b_fs_yx_fsv4},        // THREE_DIM_TRANSPOSE
+                             {{1, 4, 4, 4}, format::b_fs_yx_fsv4},        // THREE_DIM_TRANSPOSE
```

Comment:
Please add test with shapes which is not aligned with the block size

Reply:
Also can we ensure that these are selecting the new kernel? We need to enforce the implementation

Reply:
For now, this kernel implementation doesn't support leftovers, so in the "Validate" method, we check only for aligned cases.